### ISSUE TITLE
AbstractMeta::get never returns an unqualified result

### ISSUE THREAD

**Issue by JanecekPetr** [jOOQ/jOOQ#7834]
### Expected behavior and actual behavior:
In `AbstractMeta::get` (line 157) this code does nothing:

```java
List<T> list = unqualified.get(u);
if (list == null) {
    list = new ArrayList<>();
    unqualified.get(u);	// FIXME PJ: probably `unqualified.put(u, list);` intended
}

list.add(object);
```

and should probably be this:

```java
List<T> list = unqualified.get(u);
if (list == null) {
    list = new ArrayList<>();
    unqualified.put(u, list);
}

list.add(object);
```

### Steps to reproduce the problem (include example code if possible):

I did not yet come up with a testcase to support this, but it should not be too hard and I will do so if I find a little extra time today.

I could also provide a PR, but I'm sure you can manage yourself...

### Versions:

- jOOQ: master as of 2018-08-28, [d4d0fc590c320b2c5e35588a32c7a453e08eb192](https://github.com/jOOQ/jOOQ/commit/d4d0fc590c320b2c5e35588a32c7a453e08eb192)
- Java: N/A
- Database (include vendor): N/A
- OS: N/A
- JDBC Driver (include name if inofficial driver): N/A

**Comment by lukaseder** [comment]
Thanks for your report. Sharp eye! I totally agree with the fix. Will look into why this isn't covered by tests

**Comment by lukaseder** [comment]
I can reproduce the problem with a simple test case

> I could also provide a PR, but I'm sure you can manage yourself...

No worries, I'll manage.

Thanks again for your report.

**Comment by lukaseder** [comment]
Fixed for jOOQ 3.12. Backport scheduled for 3.11.5 (#7836)
